app/experimenter/static/js/components/constants.js

```diff
@@ -241,9 +241,16 @@ export const PLAYBOOK_CHOICES = [
 ];
 
 export const PLATFORM_CHOICES = [
-  { value: "All Mac", label: "All Mac" },
-  { value: "All Linux", label: "All Linux" },
-  { value: "All Windows", label: "All Windows" },
+  { value: "All Mac", label: "Mac" },
+  { value: "All Linux", label: "Linux" },
+  { value: "All Windows", label: "Windows" },
+];
+
+export const WINDOWS_VERSIONS_CHOICES = [
+  { value: "Windows 7", label: "Windows 7" },
+  { value: "Windows 8", label: "Windows 8" },
+  { value: "Windows 8.1", label: "Windows 8.1" },
+  { value: "Windows 10+", label: "Windows 10+" },
```

Comment:
I think it'd be better to move all the actual strings into individual constants and then refer to them in these arrays/objs, and then if we need to refer to an individual one (like above) you also have the individual value.  I think I found a way to serialize the `constants.py` into a json object and load it into the frontend so I think we'll be able to get rid of this file altogether soon.